Dockerfile

```diff
@@ -1,4 +1,6 @@
 FROM kuzzleio/dev
 MAINTAINER Kuzzle <support@kuzzle.io>
 
-# resolves onbuild triggers into the final image
```

Comment:
If we explicitly add the current directory on build, the ONBUILD command becomes useless on the dev container. I suppose it should be removed from it to avoid cloning the repository.